lang/python/django-simple-captcha/Makefile

```diff
@@ -9,34 +9,69 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=django-simple-captcha
 PKG_VERSION:=0.5.11
-PKG_RELEASE:=3
+PKG_RELEASE:=4
 
-PKG_MAINTAINER:=Eneas U de Queiroz <cote2004-github@yahoo.com>
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/mbi/django-simple-captcha/tar.gz/v$(PKG_VERSION)?
```

Comment:
codeload is better than PyPI for this case?

Reply:
well, I prefer tar.gz files; because they have the `strip-components`  option which makes it easier to extract files to other folder
for some reason, there are no tar.gz files on pypi: 
https://pypi.org/project/django-simple-captcha/#files

just on codeload

Reply:
:+1: :joy: